storage/txcache/monitoring.go

```diff
@@ -245,3 +245,15 @@ func (cache *TxCache) displaySendersSummary() {
 	summary := builder.String()
 	log.Info("TxCache.displaySendersSummary()", "name", cache.name, "summary\n", summary)
 }
+
+func (cache *txListForSender) monitorOnFoundInsertionPlace(iterations *int, incomingTx *WrappedTransaction) {
+	if *iterations > numIterationsSuboptimalListTraversal {
+		log.Trace("txListForSender.monitorOnFoundInsertionPlace: suboptimal", "iterations", *iterations, "nonce", incomingTx.Tx.GetNonce(), "sender", incomingTx.Tx.GetSndAddr())
+	}
+}
+
+func (cache *txListForSender) monitorOnFoundTransaction(iterations *int, txToFind *WrappedTransaction) {
+	if *iterations > numIterationsSuboptimalListTraversal {
+		log.Debug("txListForSender.monitorOnFoundTransaction: suboptimal", "iterations", *iterations, "nonce", txToFind.Tx.GetNonce(), "sender", txToFind.Tx.GetSndAddr())
```

Comment:
wondering why this has a different log level than for monitoring insertion place

Reply:
- In our Tx Generator setup, transactions do not arrive in strictly ascending order. Actually, that is leading to **massive** _wall of log_ for `monitorOnFoundInsertionPlace: suboptimal`. Will work a bit more to improve this "bad-enough-case-scenario".

 - `monitorOnFoundTransaction` is called by the "single" version of the removal - `removeTxByHash`, which is originally called from `processAndRemoveBadTransaction`. Might as well change it to **trace**, thanks.

Will fix as well, to have the same log level.